src/commands/scanner-vuln-scan.yml

```diff
@@ -0,0 +1,20 @@
+description: >
+  Run an inline vulnerability scan.
+parameters:
+  tag:
+    description: The image tag to scan
+    type: string
+  html:
+    description: Generate a vulnerability assessment in HTML format
+    type: boolean
+    default: false
+  non-os-packages:
+    description: Scan non-os packages
+    type: boolean
+    default: false
+steps:
+  - run:
+      name: Execute inline container scan
+      command: |
+        docker pull chriscircleci/vuln_django:7f7305bea884b63578c28d4513a916e2247ec152
+        lw-scanner evaluate chriscircleci/vuln_django
```

Comment:
@aedifex are these supposed to be static values here or should this be templates for the customer inputs?